server/datastore/mysql/migrations/tables/20240612154849_AddExcludeLabelToMDMProfilesAndDeclarations.go

```diff
@@ -0,0 +1,27 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240612154849, Down_20240612154849)
+}
+
+func Up_20240612154849(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE mdm_configuration_profile_labels ADD COLUMN exclude TINYINT(1) NOT NULL DEFAULT 0`)
```

Comment:
Note that the PR targets `main`, as I think we've discussed in the past that we preferred this (vs a feature branch) to avoid DB conflicts. That being said, if you feel this feature would be better to live all in the feature branch, please feel free to update the target branch and merge in my absence if needed!

Reply:
Ignore this, updated to target the feature branch.